docs/DeveloperGuide.md

```diff
@@ -1,38 +1,230 @@
+a healthier lifestyle, regardless of their knowledge in exercises or their individual fitness levels.
+Through this program, it aims to not only help users learn new workouts, while also keep track of their fitness
+progress.
+
+## User Stories
+
+| Version | As a ...                                                                                                 | I want to ...                                                                                                                   | So that I can ...                                                              |
+|---------|----------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------|
+| v1.0    | - User who wants to start working out<br/>- User who wants to train a specific part of my body           | - Select a specific intensity workout<br/>- Request a workout that comprises exercises that thoroughly exercises that body part | - Exercise according to selected intensity<br/> - Work on my target body part. |
+| V1.0    | - User                                                                                                   | - be able to filter my exercises by body part                                                                                   | - train a specific part of my body                                             |
+| V2.0    | - User                                                                                                   | - be able to start a workout and set it as complete and save it                                                                 | - reflect and track my progress                                                |
+| V2.0    | - User looking for motivation                                                                            | - be able to track my workout history as statistics                                                                             | - better visualise my overall progress                                         |
+| V2.0    | - user with little to no experience with exercise                                                        | - be given instructions for the specific exercise that I am working on                                                          | be educated on how to complete the exercise correctly                          |
+| V2.0    | - User who wants to stay motivated to workout </br> - User who wants to feel good about my past workouts | - See myself be able to accomplish or achieve incrementally greater goals </br> - Keep track of all my exercises                | - Continue to stay motivated in making exercise a fun, long-lasting habit      |

```

Comment:
![image](https://user-images.githubusercontent.com/88542282/228426549-184740c8-7d53-42ac-a406-1670926d288e.png)
Seems like some slight formatting issue here

Reply:
Yup, thats a known issue.Will correct them in the future iteration(s) 👍